lib/rubocop/cop/rootstrap.rb

```diff
@@ -0,0 +1 @@
+# require "rubocop/cop/rootstrap/custom_cop"
```

Comment:
It is right this to be commented ?

Reply:
I can remove it, it's just an example on how to require cops.

Reply:
as you are documenting in the readme I think is enough there

Reply:
Yep, makes sense

Reply:
Done